src/env.ts

```diff
@@ -164,9 +174,35 @@ export function setEnvConfiguration(config: Configuration, handlers: FunctionInf
     if (environment[ddCaptureLambdaPayloadEnvVar] === undefined) {
       environment[ddCaptureLambdaPayloadEnvVar] = config.captureLambdaPayload;
     }
+    if (type === RuntimeType.DOTNET) {
+      if (environment[ENABLE_PROFILING_ENV_VAR] === undefined) {
+        environment[ENABLE_PROFILING_ENV_VAR] = CORECLR_ENABLE_PROFILING;
+      } else if (environment[ENABLE_PROFILING_ENV_VAR] !== CORECLR_ENABLE_PROFILING) {
+        throwEnvVariableError("CORECLR_ENABLE_PROFILING");
+      }
+      if (environment[PROFILER_ENV_VAR] === undefined) {
+        environment[PROFILER_ENV_VAR] = CORECLR_PROFILER;
+      } else if (environment[PROFILER_ENV_VAR] !== CORECLR_PROFILER) {
+        throwEnvVariableError("CORECLR_PROFILER");
+      }
+      if (environment[PROFILER_PATH_ENV_VAR] === undefined) {
+        environment[PROFILER_PATH_ENV_VAR] = CORECLR_PROFILER_PATH;
+      } else if (environment[PROFILER_PATH_ENV_VAR] !== CORECLR_PROFILER_PATH) {
+        throwEnvVariableError("CORECLR_PROFILER_PATH");
+      }
+      if (environment[DOTNET_TRACER_HOME_ENV_VAR] === undefined) {
+        environment[DOTNET_TRACER_HOME_ENV_VAR] = DD_DOTNET_TRACER_HOME;
+      } else if (environment[DOTNET_TRACER_HOME_ENV_VAR] !== DD_DOTNET_TRACER_HOME) {
+        throwEnvVariableError("DD_DOTNET_TRACER_HOME");
+      }
+    }
   });
 }
 
+function throwEnvVariableError(variable: string) {
+  throw new Error(`Environment variable ${variable} not set up correctly for datadog monitoring`);
```

Comment:
What's the rationale for this error?

Reply:
I talked to Tian about this. Since we are adding Env Vars for .NET (CORECLR_ENABLE_PROFILING, CORECLR_PROFILER, etc), if the user has set, for instance `CORECLR_ENABLE_PROFILING=0` we don't want to overwrite it but we also want to call that out by erroring because if CORECLR_ENABLE_PROFILING does not equal 1, datadog won't be instrumented correctly. I'm thinking now it may be better to just add a warning in this instance but not sure since that would allow deployment of a function that isn't configured properly

Reply:
I see, I think it does make sense to throw an error here, however, as a user, I wouldn't quite understand what `Environment variable CORECLR_ENABLE_PROFILING not set up correctly for datadog monitoring` means and what I need to do to fix the issue. Could we update the error message to be a bit more specific/actionable?